src/components/Breadcrumb/index.vue

```diff
@@ -67,17 +67,19 @@ export default class extends Vue {
   private pathCompile(path: string) {
     // To solve this problem https://github.com/PanJiaChen/vue-element-admin/issues/561
     const { params } = this.$route
-    const toPath = pathToRegexp.compile(path)
+    const toPath = compile(path)
     return toPath(params)
   }
 
   private handleLink(item: any) {
+    // Throw Error "NavigationDuplicated"
+    // https://github.com/vuejs/vue-router/issues/2872
     const { redirect, path } = item
     if (redirect) {
-      this.$router.push(redirect)
+      this.$router.push(redirect).catch(_err => {})
       return
```

Comment:
Same, do you have any screenshot for the error?

Reply:
When I click the link of same destination in breadcrumb, it will throw NavigationDuplicated  in code `this.$router.push(redirect)`. I found a solution in https://github.com/vuejs/vue-router/issues/2872#issuecomment-522341874.

![image](https://user-images.githubusercontent.com/31433480/74600181-b47d1900-50c8-11ea-8b96-8016478a7974.png)

![image](https://user-images.githubusercontent.com/31433480/74600185-c6f75280-50c8-11ea-8257-4a68696dce51.png)